src/invoice_utils/config.py

```diff
@@ -16,3 +19,9 @@ def _str_to_bool(arg: str) -> bool:
 INVOICE_UTILS_MAIL_LOGIN_PASSWORD = os.getenv("INVOICE_UTILS_MAIL_LOGIN_PASSWORD")
 INVOICE_UTILS_SENDER_EMAIL = os.getenv("INVOICE_UTILS_SENDER_EMAIL")
 INVOICE_UTILS_SMTP_TLS = _str_to_bool(os.getenv("INVOICE_UTILS_SMTP_TLS"))
+INVOICE_UTILS_BODY_TEMPLATE_NAME = os.getenv("INVOICE_UTILS_BODY_TEMPLATE_NAME", DEFAULT_BODY_TEMPLATE_NAME)
+INVOICE_UTILS_BODY_TEMPLATE_PACKAGE = os.getenv("INVOICE_UTILS_BODY_TEMPLATE_PACKAGE", DEFAULT_BODY_TEMPLATE_PACKAGE)
+INVOICE_UTILS_BODY_TEMPLATE_DIRECTORY = os.getenv(
```

Comment:
Having a long, descriptive variable name is good. Having a short, descriptive variable name is better.
`INVOICE_UTILS_TEMPLATES_DIR` is a better name because:
- it designates a directory for all templates, not just the "body" templates
- it's shorter
- you still know what the variable refers to